### ISSUE TITLE
Fabric won't start by this launcher

### ISSUE THREAD

**Issue by serverchanspe** [3arthqu4ke/HeadlessMc#9]
Uncaught exception in thread "main"
net.fabricmc.loader.impl.util.ExceptionUtil$WrappedException: java.io.IOException: error reading C:\Users\mikus\AppData\Roaming\.minecraft\versions\fabric-loader-0.14.8-1.19\fabric-loader-0.14.8-1.19.jar
	at net.fabricmc.loader.impl.util.ExceptionUtil.wrap(ExceptionUtil.java:51)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.locateGame(MinecraftGameProvider.java:232)
	at net.fabricmc.loader.impl.launch.knot.Knot.createGameProvider(Knot.java:177)
	at net.fabricmc.loader.impl.launch.knot.Knot.init(Knot.java:130)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:68)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)
Caused by: java.io.IOException: error reading C:\Users\mikus\AppData\Roaming\.minecraft\versions\fabric-loader-0.14.8-1.19\fabric-loader-0.14.8-1.19.jar
	at net.fabricmc.loader.impl.game.LibClassifier.process(LibClassifier.java:214)
	at net.fabricmc.loader.impl.game.LibClassifier.process(LibClassifier.java:161)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.locateGame(MinecraftGameProvider.java:187)
	... 4 more
Caused by: java.util.zip.ZipException: zip file is empty
	at java.base/java.util.zip.ZipFile$Source.zerror(ZipFile.java:1598)
	at java.base/java.util.zip.ZipFile$Source.findEND(ZipFile.java:1382)
	at java.base/java.util.zip.ZipFile$Source.initCEN(ZipFile.java:1477)
	at java.base/java.util.zip.ZipFile$Source.<init>(ZipFile.java:1315)
	at java.base/java.util.zip.ZipFile$Source.get(ZipFile.java:1277)
	at java.base/java.util.zip.ZipFile$CleanableResource.<init>(ZipFile.java:709)
	at java.base/java.util.zip.ZipFile.<init>(ZipFile.java:243)
	at java.base/java.util.zip.ZipFile.<init>(ZipFile.java:172)
	at java.base/java.util.zip.ZipFile.<init>(ZipFile.java:186)
	at net.fabricmc.loader.impl.game.LibClassifier.process(LibClassifier.java:201)
	... 6 more

fabric-loader-0.14.8-1.19 and fabric-loader-0.14.8-1.18.2 and fabric-loader-0.14.7-1.19
Maybe There are some conflict with fabric update?

**Comment by 3arthqu4ke** [comment]
I feel like this is an issue with the fabric installer. It creates an empty jar at `C:\Users\...\AppData\Roaming.minecraft\versions\fabric-loader-0.14.8-1.19\fabric-loader-0.14.8-1.19.jar` which can't be read. Probably should be reported to them.

I was able to work around this by deleting that jar and launching the game with HeadlessMc. The proper jar will then be downloaded on launch. If you use the [HMC-Specifics](https://github.com/3arthqu4ke/HMC-Specifics/releases/tag/1.0.5) redownload them. I just updated the fabric one, it didn't work on 1.19.
